package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "qse",
+  "version": "1.0.0",
+  "description": "A cli tool to quickstart express apps",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "node bin/index.js init"
```

Comment:
Do we need these 2?

Reply:
Dev is for local testing, we can remove it, if not required